src/cast_core/CMakeLists.txt

```diff
@@ -14,46 +14,10 @@
 
 project(cast_core VERSION 0.0.1 LANGUAGES CXX)
 
-# Pacify the CLion
```

Comment:
clion upgraded the version of cmake or something so now this is no longer needed 🎉